### PR TITLE
Use everybody role instead of anonymous

### PR DESCRIPTION
**Pull request by pallix**

### CODE REVIEW

**Comment by joka** [comment]
can you merge master?

File: src/adhocracy_core/adhocracy_core/authentication/__init__.py [hunk 1]
```diff
@@ -227,7 +224,7 @@ def effective_principals(self, request: IRequest) -> list:
         if cached_principals:
             return cached_principals
         if self.authenticated_userid(request) is None:
-            principals = [Everyone, Anonymous]
+            principals = [Everyone]
```
Comment (joka) [inline src/adhocracy_core/adhocracy_core/authentication/__init__.py]: we can delete this. super().effective_principals should return [Everyone] in this case.

File: src/adhocracy_meinberlin/adhocracy_meinberlin/workflows/bplan.yaml [hunk 2]
```diff
@@ -3,15 +3,15 @@ states:
...
-      principals:                  [anonymous, participant, moderator, creator, initiator, admin]
+      principals:                  [everyone, participant, moderator, creator, initiator, admin]
       permissions:
-        - [create,                  A,         ~,           A,          A,      A,         ~]
-        - [create_proposal,         A,         ~,           ~,          ~,      ~,         ~]
-        - [edit_proposal,           ~,         ~,           ~,          ~,      ~,         ~]
+        - [create,                  A,        ~,           A,          A,      A,         ~]
+        - [create_proposal,         A,        ~,           ~,          ~,      ~,         ~]
+        - [edit_proposal,           ~,        ~,           ~,          ~,      ~,         ~]
   evaluate:
```
Comment (joka) [inline src/adhocracy_meinberlin/adhocracy_meinberlin/workflows/bplan.yaml]: If we allow Everyone to create proposal, we also allow it to authenticted user. 
But for bplan we don't want this, or?

**Comment by pallix** [comment]
@joka I merged master. For bplaene I'm waiting an answer from Gereon.

**Comment by pallix** [comment]
Gereon says both anonymous and authenticated should have access to the formular.

**Comment by joka** [comment]
> Gereon says both anonymous and authenticated should have access to the formular.

ok, thats easier

**Pull request merged**